packages/usa-button/src/test/test-patterns/test-button-links.twig

```diff
@@ -0,0 +1,14 @@
+{% if is_demo and 'usa-button--big' not in modifier %}
```

Comment:
I noticed that `usa-button.twig` [adds an additional check](https://github.com/uswds/uswds/blob/develop/packages/usa-button/src/usa-button.twig#L1-L3) to provide a dark background for the `usa-button--outline usa-button--inverse` classes. Was there a reason it was left off here? 

```suggestion
{% if 'usa-button--outline usa-button--inverse' in modifier %}
  <div class="bg-base-darkest padding-1" style="max-width: fit-content">
{% endif %}

{% if is_demo and 'usa-button--big' not in modifier %}
```

Reply:
Originally, I left it out because the modifiers have to be manually entered via text strings. Users would have to put in exactly `usa-button--outline usa-button--inverse` for it to work. So swapping them for example doesn't show the black background ` usa-button--inverse usa-button--outline`

That said, I'd rather have it so we _can_ experiment with it than not have it! 

Included the check in [814f746](https://github.com/uswds/uswds/pull/5435/commits/814f746eb4b3ae7fd75567fc088c7fc3f3bcc79d)